static/scss/answers/common/util/UtilityLayout.scss

```diff
@@ -5,93 +5,6 @@ $generate-classes: false !default;
 $col-gap: spacing(base) !default;
 $splitting-points: (sm, md, lg) !default;
 
-
-@if $generate-classes
```

Comment:
Can we get rid of the generate-classes, col-gap, and the splitting-points variables as well?

Reply:
yes, updated!